pkg/olm/installplanlist.go

```diff
@@ -0,0 +1,41 @@
+package olm
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ListInstallPlan returns list of the installplans found for specific namespace.
```

Comment:
ListInstallPlan returns a list of installplans found under a specific namespace.